mkdocs.yml

```diff
@@ -37,6 +37,7 @@ extra_css:
 
 # give a title for each page
 nav:
+  - 'Index': 'index.md'
```

Comment:
I'd prefer this be called "Home" over "Index"; index.md is currently the landing page